9.2/root/usr/bin/hooks.d/post-init

```diff
@@ -0,0 +1 @@
+#!/bin/bash
```

Comment:
Yes, wouldn't you run mysql for the queries? You'd need the database name at least, no?